src/translation/translation_fr_FR.ts

```diff
@@ -1720,17 +1720,17 @@ Nous avons coupé votre canal et activé &quot;Me silencer&quot;. Veuillez d&apo
     <message>
         <location filename="../clientsettingsdlg.cpp" line="394"/>
         <source>Audio Alerts</source>
-        <translation type="unfinished"></translation>
+        <translation>Alertes audio</translation>
     </message>
     <message>
         <location filename="../clientsettingsdlg.cpp" line="395"/>
         <source>Enable audio alert when receiving a chat message and when a new client joins the session. A second sound device may be required to hear the alerts.</source>
-        <translation type="unfinished"></translation>
+        <translation>Activez l&apos;alerte audio lors de la réception d&apos;un message de chat et lorsqu&apos;un nouveau client rejoint la session. Un deuxième périphérique sonore peut être nécessaire pour entendre les alertes.</translation>
     </message>
     <message>
         <location filename="../clientsettingsdlg.cpp" line="399"/>
         <source>Audio Alerts check box</source>
-        <translation type="unfinished"></translation>
+        <translation>Case-à-cocher alerte audio</translation>
     </message>
     <message>
         <location filename="../clientsettingsdlg.cpp" line="403"/>
```

Comment:
Where `ASIO Device Settings` is mentioned as a button in the English (line 1620), it's referring to the value below (line 1737).  The French should refer to the same translation, `Paramètres du périphérique ASIO`:
* `<location filename="../clientsettingsdlg.cpp" line="200"/>`
* `<location filename="../clientsettingsdlg.cpp" line="222"/>`

Reply:
Ok thanks @pljones. Understood. :-)